models/chain-information.ts

```diff
@@ -0,0 +1,21 @@
+export interface ChainInformation {
+  chainName: string
+  description: string
+  platformChainId: string
+  subnetId: string
+  vmId: string
+  vmName: string // EVM, BITCOIN, ETHEREUM
+  rpcUrl: string // uri
+  wsUrl?: string // uri
+  multicallAddress?: string // ChecksumAddress
+  isTestnet: boolean
+  networkToken: {
+    name: string
+    symbol: string
+    description: string
+    decimals: number
+    coingeckoId?: string
+  }
+  coingeckoAssetPlatformId?: string
```

Comment:
For both `coingeckoId` and `coingeckoAssetPlatformId` I think we should give ourselves options for extensibility when it comes to pricing providers.

Think it makes sense to provide a `pricingProviders`/`pricingProviderConfig` item at the top-level that can contain provider configuration for different providers.

One of those should be coingecko but may be others in the future.

The `ConigeckoConfig` type can contain two optional fields for `coingeckoId` and `assetPlatformId`.

This is more or less what we have done on the glacier side too.  WDYT?

Reply:
For context:
https://github.com/ava-labs/glacier-api/blob/8e8e7a2b557a35945295153905c63d46f9d2e46e/src/common/interfaces/configuration.ts#L17